tools/run_pr_builder.sh

```diff
@@ -18,7 +18,17 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
 go build ${GO_LIST}
-go test ${GO_LIST}
+
+# go test plus build coverage data for upload codecov.io
+echo "" > coverage.txt
+for d in ${GO_LIST}; do
+    go test -coverprofile=profile.out -covermode=atomic $d
+    if [ -f profile.out ]; then
+        cat profile.out >> coverage.txt
+        rm profile.out
+    fi
+done
+bash <(curl -s https://codecov.io/bash) -t ${COVERALLS_TOKEN}
```

Comment:
No. The badge shows results for the master branch - it is hard-coded - not updating the README.md links based on the git branch or PR. If you go to codecov.io, you can see the PRs listed and select the one you are reviewing or working on. I will also setup the Slack notifications once I have more data and verify it isn't too noisy.